Formula/elasticsearch@6.rb

```diff
@@ -14,8 +14,7 @@ class ElasticsearchAT6 < Formula
 
   keg_only :versioned_formula
 
-  depends_on arch: :x86_64 # openjdk@8 is not supported on ARM
-  depends_on "openjdk@8"
+  depends_on "openjdk"
```

Comment:
Please bump the `revision` (either add `revision 1` or increment the `revision` right after `license`) so that users who have this installed get the updated version of the formula when they do `brew upgrade`.